backend/src/core/files.py

```diff
@@ -0,0 +1,42 @@
+import os
+import secrets
+
+import aiofiles
+from fastapi import UploadFile
+from fastapi.responses import FileResponse
+
+from src.core.config import settings
+
+
+def generate_name(file: UploadFile):
```

Comment:
Rename it to `generate_unique_name` for clarity

Reply:
Done